package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "gpii-devpmt",
+    "productName": "devpmt",
+    "version": "1.0.0",
+    "description": "Developers Preference Testing Tool",
+    "main": "standalone.js",
+    "scripts": {
+        "start": "electron standalone.js",
+        "watch-sass": "gulp",
+        "build-sass": "gulp sass"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/sgithens/gpii-devpmt"
+    },
+    "author": "Steven Githens <sgithens@raisingthefloor.org>",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://issues.gpii.net/"
+    },
+    "homepage": "https://github.com/sgithens/gpii-devpmt",
+    "dependencies": {
+        "electron": "1.4.1",
+        "foundation-sites": "6.3.0",
+        "gpii-express": "1.0.2",
+        "gpii-handlebars": "the-t-in-rtf/gpii-handlebars#GPII-2474",
```

Comment:
This can probably be updated to the 1.1.0 release of gpii-handlebars

Reply:
There may be some small work required to migrate to 1.1.0, but 1.0.2 includes that merged branch.

Reply:
This is pinned to 1.1.0 now.  I don't think I actually see anything breaking from this change.